Sources/LeafKit/Data/TemplateDataEncoder.swift

```diff
@@ -1,3 +1,5 @@
+// TODO: - Should this be reimplemented? This let Leaf3 use Futures as data types?
+
```

Comment:
General question - On Discord Caleb asked about whether Leaf3's taking futures as LeafData would be added back - is that what this file was from?

Reply:
Noted per discord convo as *not* likely to be re-introduced